### ISSUE TITLE
Result

### ISSUE THREAD

**Issue by bnulihaixia** [dichotomies/proxy-nca#2]
Do you reach the performance as the paper listed on CUB-200-2011?

**Comment by dichotomies** [comment]
Hi, I'm not reaching the performance (yet - still fine-tuning), more like half of it.

**Comment by bnulihaixia** [comment]
2 problems should be mentioned:
1. the pre-trained model used in the paper is inception-BN(inception v2), not inception-v3.
2. data transform is very import.



PS: I have make a Pytorch project for Deep Metric Learning, and reach good performance. You can fork it and modified to  Proxy-NCA.

**Comment by dichotomies** [comment]
Hey sorry for the late replay, was very busy the last days.

Forked your repository 3 days ago instantly after seeing your great work. I'm currently looking into it.

Yes, would be better to use the original version for comparison.

Could you please elaborate on the data transform part? I'm currently using: resize, random crop and normalization (mean, std).

**Comment by bnulihaixia** [comment]
I am very Sorry, I made a mistake. 
What I really want to say is that how to sampling data to constitute minibatch is very important.  Not data transform.

**Comment by dichotomies** [comment]
Oh, that sounds even more interesting. Could you please elaborate further on that? I can't find something about it in the paper (I'm new to metric learning).

**Comment by bnulihaixia** [comment]
Sorry. I also don't know that how the paper samples minibatch.
I just think this maybe a reason why your  implement has not reached similar performance as the paper said.
If you want to know their sampling way, you should ask the author.

**Comment by dichotomies** [comment]
Hi, agree on that it's important on how to sample the mini batches in the usual triplet metric learning methods. But I think that one main contribution of this paper is a solution to this problem. On page 3 they state:

> To address the above sampling problem, we propose to learn a small set of data points P with |P| ≪ |D|.

Correct me please if I'm wrong.

**Comment by dichotomies** [comment]
To answer the actual question: You can find the results for CUB and the script for reproducing them in the updated repo. The model used for training is Inception V2 (like in the paper).
